examples/defaulter-gen/generators/defaulter.go

```diff
@@ -144,6 +144,9 @@ type defaulterFuncMap map[*types.Type]defaults
 
 // Returns all manually-defined defaulting functions in the package.
 func getManualDefaultingFunctions(context *generator.Context, pkg *types.Package, manualMap defaulterFuncMap) {
+	if pkg == nil {
```

Comment:
...why is the caller sending a nil `pkg`?

Reply:
It's been a while and I forget why, but I think I found it when I was trying to regenerate during the rebase PR process.